sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LanguageServiceClient.cs

```diff
@@ -1286,7 +1286,8 @@ private static HealthcareLROTask CreateHealthcareTask(AnalyzeHealthcareEntitiesO
                 {
                     ModelVersion = options.ModelVersion,
                     StringIndexType = Constants.DefaultStringIndexType,
-                    LoggingOptOut = options.DisableServiceLogs
+                    LoggingOptOut = options.DisableServiceLogs,
```

Comment:
Does this need to be reflected to the Legacy API?

Reply:
No. it is an addition that was introduced only in the Language service.
There is no equivalent on where to assign it if the user passes a previous service version. If they still assign the value and target a previous version, the SDK drops the property

Reply:
Is this the first case of us hitting the "how do we handle mismatch errors?"    Does the behavior change or trigger anything else strange for the legacy surface?    Should we drop note in #28060 to mention this scenario for follow-up?

Reply:
> Is this the first case of us hitting the "how do we handle mismatch errors?" 

It indeed is :)

>  Does the behavior change or trigger anything else strange for the legacy surface?

It does not. The `FhriVersion` input property only affects the `FhirBundle` output property which was introduced in this new version too.

>Should we drop note in https://github.com/Azure/azure-sdk-for-net/issues/28060 to mention this scenario for follow-up?

Added. I'm following what we have done in this library when a new property is added that is only supported in specific versions and does not affect behavior of older version output properties:
- We add that information to docstrings for both input and output property.
- Input property:
  - If the property is a query parameter, we send it to the service and let them manage it.
  - If the property is a body parameter, we drop it.
- Output property: Value is always empty Dict even when targeting older versions (I added comments in the tests to pint this out)